controllers/pipeline/pipeline_adapter_test.go

```diff
@@ -82,12 +84,23 @@ var _ = Describe("Pipeline Adapter", Ordered, func() {
 		deploymentTargetClaim      *applicationapiv1alpha1.DeploymentTargetClaim
 		snapshotEnvironmentBinding *applicationapiv1alpha1.SnapshotEnvironmentBinding
 		integrationTestScenario    *integrationv1alpha1.IntegrationTestScenario
+
+		SampleImage              string
+		SampleDigest             string
+		SampleImageWithoutDigest string
 	)
+
 	const (
 		SampleRepoLink = "https://github.com/devfile-samples/devfile-sample-java-springboot-basic"
 		SampleCommit   = "a2ba645d50e471d5f084b"
 	)
 
+	SampleImageWithoutDigest = "quay.io/redhat-appstudio/sample-image"
+	imageRef, _ := name.ParseReference("quay.io/redhat-appstudio/sample-image" + ":test-index-pass")
+	imageDesc, _ := remote.Get(imageRef)
+	SampleDigest = imageDesc.Digest.String()
+	SampleImage = SampleImageWithoutDigest + "@" + SampleDigest
+
```

Comment:
Shall we have this in `const`?

Reply:
After the discussion above, since we don't need to check if the image is valid, I think I can add it to `const` and use a valid sha string here. Thank you!